syntax = "proto3";

package fraud;

service FraudDetectionService {
    rpc DetectFraud (FraudDetectionRequest) returns (FraudDetectionResponse);
}

message FraudDetectionRequest {
    string card_number = 1;
    float order_amount = 2;
}

message FraudDetectionResponse {
    bool is_fraud = 1;
}
   
